ignite/metrics/epoch_metric.py

```diff
@@ -157,8 +157,14 @@ def compute(self) -> float:
             result = self.compute_fn(_prediction_tensor, _target_tensor)
 
         if ws > 1:
-            # broadcast result to all processes
-            result = cast(float, idist.broadcast(result, src=0))
+            if isinstance(result, tuple):
+                l = len(result)
+                for i in range(l):
+                    result[i] = idist.broadcast(result[i], src=0)
```

Comment:
Why not adding the cast ?

Reply:
It is already a pytorch tensor, so I couldn't figure out what to cast to. Sorry for the delay, was caught up in a few works.